.gitmodules

```diff
@@ -104,3 +104,10 @@
 [submodule "tools/gyp"]
 	path = tools/gyp
 	url = https://chromium.googlesource.com/external/gyp
+# For building a development apache.
```

Comment:
How much data are you adding here? I guess it's pretty minor compared to apache?